bridge/github/config_test.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSplitURL(t *testing.T) {
+	type args struct {
+		url string
+	}
+	type want struct {
+		owner   string
+		project string
+		err     error
+	}
+	tests := []struct {
+		name string
+		args args
+		want want
+	}{
+		{
+			name: "default url",
+			args: args{
+				url: "https://github.com/MichaelMure/git-bug",
+			},
+			want: want{
+				owner:   "MichaelMure",
+				project: "git-bug",
+				err:     nil,
+			},
+		},
+
+		{
+			name: "default url with git extension",
+			args: args{
+				url: "https://github.com/MichaelMure/git-bug.git",
+			},
+			want: want{
+				owner:   "MichaelMure",
+				project: "git-bug",
+				err:     nil,
+			},
+		},
+		{
+			name: "url with git protocol",
+			args: args{
+				url: "git://github.com/MichaelMure/git-bug.git",
+			},
+			want: want{
+				owner:   "MichaelMure",
+				project: "git-bug",
+				err:     nil,
+			},
+		},
+		{
+			name: "ssh url",
+			args: args{
+				url: "git@github.com:MichaelMure/git-bug.git",
+			},
+			want: want{
+				owner:   "MichaelMure",
+				project: "git-bug",
+				err:     nil,
+			},
+		},
+		{
+			name: "bad url",
+			args: args{
+				url: "https://githb.com/MichaelMure/git-bug.git",
+			},
+			want: want{
+				err: ErrBadProjectURL,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			owner, project, err := splitURL(tt.args.url)
+			assert.Equal(t, tt.want.err, err)
+			assert.Equal(t, tt.want.owner, owner)
+			assert.Equal(t, tt.want.project, project)
+		})
+	}
+}
+
+func TestValidateProject(t *testing.T) {
+	tokenPrivateScope := os.Getenv("GITHUB_TOKEN_PRIVATE")
```

Comment:
I'm not sure about these environment variable names. We already use one with name `GITHUB_TOKEN`in `import_test.go`, should we rename it to `GITHUB_TOKEN_PRIVATE` too ?

Reply:
yes, let keep all that consistent, we only need two kind of tokens

Reply:
Done !